app/models/household_member.rb

```diff
@@ -92,13 +92,6 @@ class HouseholdMember < ApplicationRecord
     ["Other", "other_relation"],
   ].freeze
 
-  TAX_RELATIONSHIP_LABELS_AND_KEYS = [
```

Comment:
huh, were these just not used anymore?

Reply:
Yup, originally these were used in a dropdown, but we changed the UI later to use radio button sets that discern between spouses and dependent/other.